sk0-front/src/app/auth/register/register.page.ts

```diff
@@ -0,0 +1,59 @@
+import { Component, OnInit } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { FormsModule } from '@angular/forms';
+import { IonicModule } from '@ionic/angular';
+import { AlertController } from '@ionic/angular';
+
+@Component({
+  selector: 'app-register',
+  templateUrl: './register.page.html',
+  styleUrls: ['./register.page.scss'],
+  standalone: true,
+  imports: [IonicModule, CommonModule, FormsModule]
+})
+export class RegisterPage {
+  name: string = '';
+  email: string = '';
+  password: string = '';
+
+  constructor(public alertController: AlertController) {}
+
+  async register() {
+    if (!this.name || !this.email || !this.password) {
+      this.errorAlert('Minden mező kitöltése kötelező.');
+      return;
+    }
+
+    const emailValid = /^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}$/;
```

Comment:
Should we create these regexp values with every event or should we rather move it to class level?